io/io/src/TStreamerInfo.cxx

```diff
@@ -257,13 +257,14 @@ namespace {
    {
       std::string localtypename(s->GetUnderlyingTypeName());
       Int_t ndim = 0;
-      Int_t totaldim = 1;
+      Int_t totaldim = 0;
```

Comment:
Perhaps merge this with the commit adding `totaldim`

Reply:
Done.